include/glow/Quantization/Base/Base.h

```diff
@@ -76,6 +76,10 @@ enum Schema {
 /// parameters \p TQP.
 int8_t quantize(float input, const TensorQuantizationParams &TQP);
```

Comment:
could you make this templated to support `int8` and `int32`.

Reply:
@rdzhabarov 
I tried to make it templated like:
```
template <class DestTy = int8_t>
inline DestTy quantize(float input, const TensorQuantizationParams &TQP);
```
But the release build has following error: 
```
Undefined symbols for architecture x86_64:
  "signed char glow::quantization::quantize<signed char>(float, glow::TensorQuantizationParams const&)", referenced from:
      glow::OpenCLFunction::execute() in libOpenCL.a(OpenCL.cpp.o)
      glow::LLVMIRGen::generateLLVMIRForDataParallelInstr(llvm::IRBuilder<llvm::ConstantFolder, llvm::IRBuilderDefaultInserter>&, glow::Instruction const*, llvm::Function*, llvm::DenseMap<glow::Value*, int, llvm::DenseMapInfo<glow::Value*>, llvm::detail::DenseMapPair<glow::Value*, int> >&, llvm::Value*) in libCPUBackend.a(LLVMIRGen.cpp.o)
      glow::InterpreterFunction::fwdSplatInst(glow::SplatInst const*) in libInterpreter.a(InterpreterNodes.cpp.o)
      glow::InterpreterFunction::fwdElementMaxInst_I8Impl(glow::ElementMaxInst const*) in libInterpreter.a(InterpreterNodes.cpp.o)
      glow::InterpreterFunction::fwdElementMinInst(glow::ElementMinInst const*) in libInterpreter.a(InterpreterNodes.cpp.o)
      glow::InterpreterFunction::fwdQuantizeInst(glow::QuantizeInst const*) in libInterpreter.a(InterpreterNodes.cpp.o)
      glow::InterpreterFunction::fwdRescaleQuantizedInst(glow::RescaleQuantizedInst const*) in libInterpreter.a(InterpreterNodes.cpp.o)
      ...
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```